### ISSUE TITLE
Question about installation

### ISSUE THREAD

**Issue by konstantinosDMS** [Snorby/snorby#497]
Can I have snort in a sensor and snorby in another pc running and collect data via barnyard?

**Comment by topper727** [comment]
Yes, I done it. Trying to reset it now. It will work. I used PFsense with snort on there and had snorby on proxmox server and it worked great

**Comment by jkellogg90** [comment]
I've got this working. Snort and Barnyard2 running on an RPI and logging events to a remote VM with MySQL and Snorby.

https://github.com/bensooter/Snort16OnUbuntu
https://sublimerobots.com/2015/12/snort-2-9-8-x-on-ubuntu-part-7c/
